app/assets/javascripts/app/error_tracking/error_tracking.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('council.error_tracking', [])
+    .run(errorTracking);
```

Comment:
i like the fact that it is a separate module. I even think that it could even drop the `council` prefix.
This way it's just plug and play (download and inject) into the main app. any app really.
it makes sense for me this way. Although run blocks are difficult to test, I like it.